.github/workflows/create-sentry-release.yml

```diff
@@ -0,0 +1,26 @@
+name: Create Sentry release
+on:
+    repository_dispatch:
+        types: [sentry-release]
+
+jobs:
+    create-sentry-release:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+              with:
+                  fetch-depth: 0
+            - name: Build
+              run: yarn workspace frontend build
```

Comment:
Do we need a yarn install before the build ?

Reply:
thanks for spotting this! I believe so.